### ISSUE TITLE
Question about 2D supervision

### ISSUE THREAD

**Issue by rayeeli** [pmj110119/RenderOcc#21]
Hi author, thanks for your great work and open-source code! I have a question related to supervision. RenderOcc uses depth loss and semantic loss, but why not add RGB loss? Have you experimented with RGB loss? 

Thank you for your attention!

**Comment by pmj110119** [comment]
In the early stages of our experiments, we attempted RGB supervision, but it did not yield benefits, so we did not pursue it further.

I can provide some speculations for your consideration:

(1) Predicting RGB values is much more challenging than predicting semantics. **For the 3D-Semantic-Occ benchmark, which is semantic-centric**, introducing RGB supervision in the presence of existing semantic labels brings limited gains and makes network optimization more difficult.

However, in scenarios lacking semantic labels or when aiming to simultaneously accomplish other tasks such as scene RGB-reconstruction, leveraging rich and additional RGB supervision becomes necessary.

(2) There is significant **variation in ISP result of different cameras** in nuScenes, leading to **ambiguity in RGB values** for the same object across different cameras, affecting network optimization. As shown in below:

![1701100048687](https://github.com/pmj110119/RenderOcc/assets/40192364/07298d15-8483-49a9-86bd-a26689aab84b)


While position embedding based on `coordinates` and `viewing direction` seems to be a solution to this issue, practical testing has revealed limited effectiveness. Since we need to optimize Semantic-Density-Field cross-scene rather than per-scene overfitting, these ambiguities may have a significant impact on the model's convergence.

Designing specific embeddings to assist the model in optimization might be a way to try.
